src/stories/components/SectionTitle/SectionTitle.js

```diff
@@ -0,0 +1,84 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Fragment } from 'react';
+
+/**
+ * Component for Section Title element.
+ *
+ * @component
+ * @param {string} modifierClasses Class modifiers of the component.
+ * @param {string} url URL of the component.
+ * @param {string} imagePath The path to the img for this component.
+ * @param {string} title The title of this component.
+ * @param {string} paragraph The paragraph in this component.
+ * @param {string} svgClass A class to determine which SVG component uses.
+ * @return {object} (
+ *   <SectionTitle modifierClasses={modifierClasses} url={url}
+ *      imagePath={imagePath} title={title}
+ *      paragraph={paragraph} svgClass={svgClass} />
+ * )
+ */
+export const SectionTitle = ({
+  modifierClasses, url, imagePath, title, paragraph, svgClass,
+}) => {
+  const classes = ['section-title', `${modifierClasses}`].join(' ').trim();
+  const svgClasses = ['section-title__icon', `${svgClass}`].join(' ').trim();
+
+  return (
+    <Fragment>
```

Comment:
@rileyhgrant don't need `Fragment` if you have a wrapping `div`.

Reply:
Ah, right, that makes sense. Thanks!